README.md

```diff
@@ -56,9 +56,7 @@ func main() {
 	// return a non-empty packet list even if there is an error. This is
 	// intended, because the transaction may fail at any point, and we
 	// still want to know what packets were exchanged until then.
-	// The `nil` argument indicates that we want to use a default Solicit
-	// packet, instead of specifying a custom one ourselves.
-	conversation, err := client.Exchange("eth0", nil)
+	conversation, err := client.Exchange("eth0")
```

Comment:
Can you explain here that it will build a "default" solicit message, and that the user can override it via modifiers?